react/package.json

```diff
@@ -4,12 +4,21 @@
   "private": true,
   "license": "UNLICENSED",
   "dependencies": {
-    "react": "^16.13.0"
+    "classnames": "^2.2.6",
+    "react": "^16.13.0",
+    "react-apollo": "^3.1.3",
+    "react-intl": "^3.12.0"
   },
   "devDependencies": {
+    "@types/classnames": "^2.2.9",
     "@types/graphql": "^14.5.0",
     "@types/jest": "^25.1.3",
     "@types/node": "^13.7.7",
-    "@types/react": "^16.9.23"
+    "@types/react": "^16.9.23",
+    "vtex.checkout-graphql": "https://lucas--checkoutio.myvtex.com/_v/private/typings/linked/v1/vtex.checkout-graphql@0.23.0+build1582915320/public/@types/vtex.checkout-graphql",
+    "vtex.checkout-resources": "https://lucas--checkoutio.myvtex.com/_v/private/typings/linked/v1/vtex.checkout-resources@0.19.0+build1582915328/public/@types/vtex.checkout-resources",
```

Comment:
These depend on your workspace. You should either remove them or rerun `vtex setup` without no app linked.

Reply:
just updated the packages!